### DISCUSSION TITLE
Is anyone opposed to switching from Dependabot to Renovate?

### DISCUSSION THREAD

**Discussion by hawkrives** [frog-pond/ccc-server#713] [Polls]
As per the title. 

In general, we've mostly re-standardized on Renovate over Dependabot, as Dependabot has been nearly stagnant since GitHub bought them, and Renovate has that nice Dependency Dashboard it maintains.

**Comment by hawkrives** [comment]
Done in https://github.com/frog-pond/ccc-server/pull/758

**Reply by drewvolz** [reply]
Thank you!
